### PR TITLE
Add Artifact Hub

### PR DESCRIPTION
**Pull request by aabouzaid**
artifacthub

**Issue:** #6116
**Alexa rank:** 171,060

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

- I've re vectorized the logo with the same dimensions because the original logo is just a stroke!
- The color is the main color from the [Artifact Hub](https://artifacthub.io/) website.

### CODE REVIEW

**Comment by aabouzaid** [comment]
@service-paradis Anything is still needed to merge this PR?

**Comment by service-paradis** [comment]
> @service-paradis Anything is still needed to merge this PR?

Nothing from you for now. The missing part is the review from one of the project maintainers. We've all been quite busy for the past weeks, but this PR turns will come. During the review process, we'll let you know if some changes are required. If not, your PR will be merged.

**Comment by aabouzaid** [comment]
@service-paradis all good then, thanks for the info :+1:

File: _data/simple-icons.json
Comment (service-paradis) [inline _data/simple-icons.json:794]: I found the source in their Github repo. Can you change it for:
https://github.com/artifacthub/hub/blob/537e4862d0aa0a689eaa66e06354c1fb84eaf134/web/public/static/media/logo/artifacthub-brand-white.svg

File: slugs.md
Comment (service-paradis) [inline slugs.md:166]: Can you revert this change? This file is generated automatically on release.

File: icons/artifacthub.svg
Comment (service-paradis) [inline icons/artifacthub.svg:1]: When comparing to the sorce SVG, I see some differences. Ex.
image

It might be due to the SVGO precision. Maybe you will need to try with a precision of 4. It might be due to the stroke to path conversion too. Can you take a look at it and gave this another try?

**Review by service-paradis — CHANGES_REQUESTED** [review]
Hello @aabouzaid, thank you for your contribution!!

I made some comments. Can you take a look?

File: icons/artifacthub.svg
Comment (aabouzaid) [reply icons/artifacthub.svg]: @service-paradis I did some changes in the logo and also used a precision of 4 in SVGO ... it looks better now even it's not 100% match.

Comment (service-paradis) [reply icons/artifacthub.svg]: Thank you for your work. As you said, it is much better!

Tools are not really good to convert stroke to path and this icon id more difficult than it appear!
Still, we really try to have a perfect fit with the original and there are some differences that we can ignore.

I gave it a try and got the following path. It might need some adjustments to have a better fit, to optimize and to pass the linter, but you can use it as a starting point if you think it can help you.

```
m 11.999414,24.000438 c -0.616974,0.0012 -1.24203,-0.172158 -1.78,-0.5 L 2.71,19.208 C 1.6924162,18.591195 1.0700559,17.490364 1.0694135,16.300438 V 7.7304382 c 0,-1.253 0.6369062,-2.369375 1.76,-3.01 L 10.25,0.48 c 1.079982,-0.63864911 2.415018,-0.63764911 3.495,0.001 l 7.419414,4.2374382 c 1.080027,0.5961014 1.771323,1.7340139 1.762586,3.01 V 16.31 c 0,0.967687 -0.443367,2.163782 -1.636586,2.898438 l -7.510414,4.292 c -0.563437,0.343937 -1.198555,0.5 -1.781586,0.5 z m -0.5,-21.3459998 L 3.955,6.963 C 3.7544063,7.1196875 3.5759541,7.3321221 3.576,7.7304382 L 3.577,16.400438 C 3.6145313,16.620875 3.6959063,16.8285 3.95,17.035 l 7.559414,4.316438 c 0.266,0.172 0.664,0.21 0.985,0 L 20.012,17.055 c 0.231719,-0.148438 0.371,-0.388562 0.41,-0.653562 V 7.7304382 C 20.4315,7.431606 20.286058,7.1418659 20.045,6.965 L 12.499414,2.6534382 c -0.390562,-0.2393063 -0.759172,-0.1557376 -1,10e-4 z
```

Comment (aabouzaid) [reply icons/artifacthub.svg]: @service-paradis this looks great, I've updated the file after small adjustments (set the height to 24px and set precision to 5)
Thanks for your help :bow:

**Pull request merged**
